packages/kbn-typed-react-router-config/src/create_router.ts

```diff
@@ -122,7 +125,15 @@ export function createRouter<TRoutes extends RouteMap>(routes: TRoutes): Router<
         );
 
         if (isLeft(decoded)) {
-          throw new Error(PathReporter.report(decoded).join('\n'));
+          return {
```

Comment:
@dgieselaar WDYT of not throwing an exception here anymore?

Reply:
Why is it necessary? Can we perhaps throw an error with the needed metadata?

Reply:
I rollbacked the way it was before